src/main/resources/application-dev.yaml

```diff
@@ -128,19 +128,19 @@ containers:
         - 12625
         - 12624
       ocrValidationUrl: ${OCR_VALIDATION_URL_PROBATE}
-      paymentsEnabled: true
+      paymentsEnabled: {$PAYMENTS_ENABLED_PROBATE}
```

Comment:
I think we can be setting a default value of false with these new environment variables. So they are like the other binary ones in the application.yaml such as ${NOTIFICATIONS_TO_ORCHESTRATOR_TASK_ENABLED:false}, ${SCAN_ENABLED:false}, ${LAUNCH_DARKLY_OFFLINE_MODE:false} etc.

Reply:
> added changes

It looks like some are default to true. All should default to false. Basically what we are saying by having all default as false is: if the payment enabled variable is missing/not set, then do not allow payments to be enabled.